.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -95,14 +97,17 @@ jobs:
     - name: Pull legacy checkpoints
       run: bash .actions/pull_legacy_checkpoints.sh
 
-    - name: Install dependencies
-      env:
-        PACKAGE_NAME: pytorch
-        FREEZE_REQUIREMENTS: 1
+    - name: Adjust PyTorch versions in requirements files
       run: |
-        # adjust PyTorch versions in requirements files
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt ${{ matrix.pytorch-version }}
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/examples.txt ${{ matrix.pytorch-version }}
+        cat requirements/pytorch/base.txt
+      shell: bash
+
+    - name: Install package & dependencies
+      env:
+        PACKAGE_NAME: pytorch
+      run: |
         # install PL and optional dependencies for testing
         pre_option=$(python -c "print('--pre' if '${{matrix.release}}' == 'pre' else '')" 2>&1)
```

Comment:
I believe this logic is making a different package fail. `--pre` should only be applied to torch. In fact, do we need to set this flag if we area already pulling from `torch_test.html`?